### PR TITLE
fix: trivial e2e improvements

### PR DESCRIPTION
**Pull request by toteki**
Anything that doesn't need much review

- adding comments to functions
- moving functions between files without modifying
- merging short, single-use functions into the functions that call them

This way the main PR is less cluttered

### CODE REVIEW

File: tests/e2e/setup/chain.go [hunk 1]
```diff
@@ -56,64 +57,44 @@ func newChain() (*chain, error) {
...
+		// TODO (comment): which are we doing? loading or generating?
 		if err := node.createConsensusKey(); err != nil {
 			return err
 		}
-	}
-
-	return nil
-}
 
-func (c *chain) createAndInitGaiaValidator(cdc codec.Codec) error {
```
Comment (toteki) [inline tests/e2e/setup/chain.go:89]: moved to setup/gaia.go

File: tests/e2e/setup/validator.go [hunk 2]
```diff
@@ -40,21 +40,25 @@ type validator struct {
...
 	return fmt.Sprintf("%s%d", v.moniker, v.index)
 }
 
+// configDir is <chain.configDir()>/<val.instanceName>
 func (v *validator) configDir() string {
 	return fmt.Sprintf("%s/%s", v.chain.configDir(), v.instanceName())
 }
 
-func (v *validator) createConfig() error {
+// // createConfigDir makes the validator's config directory at <chain.configDir()>/<val.instanceName>/config
```
Comment (kosegor) [inline tests/e2e/setup/validator.go:53]:
```suggestion
// createConfigDir makes the validator's config directory at <chain.configDir()>/<val.instanceName>/config
```

**Review by kosegor — APPROVED** [review]
LGTM!

File: tests/e2e/e2e_oracle_test.go [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+	s.Require().NoError(err)
+}
+
+func (s *E2ETest) TestUpdateOracleParams() {
+	params, err := s.Umee.QueryOracleParams()
+	s.Require().NoError(err)
+
+	s.Require().Equal(uint64(5), params.HistoricStampPeriod)
+	s.Require().Equal(uint64(4), params.MaximumPriceStamps)
+	s.Require().Equal(uint64(20), params.MedianStampPeriod)
```
[the remark this answers is not in the document]
Comment (toteki) [reply tests/e2e/e2e_oracle_test.go:24]: It could be - I merely moved this function between files without modifying it

**Pull request merged**